wherehows-web/app/constants/notifications.ts

```diff
@@ -3,3 +3,9 @@
  * NOTE: This value corresponds with animation speed value set in styles/abstracts/_variables.scss
  */
 export const bannerAnimationSpeed = 0.6;
+
+/**
+ * Base message for 2 factor auth banner message for the login screen.
+ */
+export const showTwoFABannerMessage =
```

Comment:
Minor, drop the `show` from `showTwoFABannerMessage ` since it's not a boolean flag